### PR TITLE
Tests for expiring first party tracking cookies

### PR DESCRIPTION
**Pull request by kdzwinel**
tech design: https://app.asana.com/0/0/1201277458549588/f

### CODE REVIEW

File: expire-first-party-tracking-cookies/tests.json [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+                "name": "below treshold - expiry not modified",
+                "siteURL": "https://example.com/",
+                "scriptURL": "https://tracker.com/script.js",
+                "setDocumentCookie": "foo=bar; Max-Age=1000; Domain=example.com; Path=/; Secure",
+                "expectCookieSet": true,
+                "expectExpiryToBe": 1000,
+                "exceptPlatforms": []
+            },
+            {
+                "name": "above treshold - expiry modified",
```
Comment (sammacbeth) [inline expire-first-party-tracking-cookies/tests.json:16]:
```suggestion
                "name": "above threshold - expiry modified",
```

File: expire-first-party-tracking-cookies/tests.json [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+            },
+            {
+                "name": "same entity - expiry not modified",
+                "siteURL": "https://tracker.test/",
+                "scriptURL": "https://tracker.com/script.js",
+                "setDocumentCookie": "foo=bar; Max-Age=900000; Domain=tracker.test; Path=/; Secure",
+                "expectCookieSet": true,
+                "expectExpiryToBe": 900000,
+                "exceptPlatforms": [
+                    "web-extension"
```
Comment (sammacbeth) [inline expire-first-party-tracking-cookies/tests.json:41]: I'll get this one fixed :)

**Review by sammacbeth — APPROVED** [review]
Looks good - just some typos in the test names, and I'll fix the same entity tests so that can be re-enabled.

File: expire-first-party-tracking-cookies/tests.json [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+                "scriptURL": "https://not-a-tracker.com/script.js",
+                "setDocumentCookie": "foo=bar; Max-Age=900000; Domain=example.com; Path=/; Secure",
+                "expectCookieSet": true,
+                "expectExpiryToBe": 900000,
+                "exceptPlatforms": []
+            },
+            {
+                "name": "above treshold - cnamed tracking script - expiry modified",
+                "siteURL": "https://example.com/",
+                "scriptURL": "https://tracker.cname.com/script.js",
```
Comment (sammacbeth) [inline expire-first-party-tracking-cookies/tests.json:74]: Could you also test scripts cnamed to same eTLD+1 as the site?

Comment (kdzwinel) [reply expire-first-party-tracking-cookies/tests.json:74]: good idea, added!

**Comment by kdzwinel** [comment]
Thanks some! All comments addressed, new test is passing in extension.

**Pull request merged**